src/Metadata/Resource/Factory/InputOutputResourceMetadataFactory.php

```diff
@@ -64,6 +64,15 @@ private function getTransformedOperations(array $operations, array $resourceAttr
 
             $operation['input'] = isset($operation['input']) ? $this->transformInputOutput($operation['input']) : $resourceAttributes['input'];
             $operation['output'] = isset($operation['output']) ? $this->transformInputOutput($operation['output']) : $resourceAttributes['output'];
+
+            if (
+                !isset($operation['status'])
+                && isset($operation['output'])
+                && \array_key_exists('class', $operation['output'])
```

Comment:
```suggestion
                && \array_key_exists('class', $operation['output'] ?? [])
```

and drop the isset before ?

Reply:
I'd be to keep isset, it's readable and doesn't harm no?

Reply:
ok